### PR TITLE
Move Refiner and Sort Documents from beta documents to V1 documents

### PR DESCRIPTION
**Pull request by yiwenwang-ms**
Ready to move refiner and sort from beta endpoint to V1 endpoint, need to synchronize refiner and sort documents from beta to v1.

### CODE REVIEW

File: api-reference/v1.0/resources/search-api-overview.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ Search requests run on behalf of the user. Search results are scoped to enforce
 |[Get the most relevant emails](#get-the-most-relevant-emails) | **enableTopResults** |
 |[Get selected properties](#get-selected-properties) | **fields** |
 |[Use KQL in query terms](#keyword-query-language-kql-support) | **query** |
-
+|[Sort search results](#sort-search-results)| **sort** |
+|[Refine results using aggregations](#refine-results-using-aggregations)|
```
Comment (ZChristine) [inline api-reference/v1.0/resources/search-api-overview.md:32]: double check whether "**aggregations** |" will be put in the below column

**Review by ZChristine — APPROVED** [review]
Looking good. i leave comments that should take care for

File: api-reference/v1.0/resources/search-api-overview.md (hunk 1 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/search-api-overview.md]: Thanks for your kindly review, resolved.

File: api-reference/v1.0/resources/aggregationoption.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+author: "nmoreau"
+ms.prod: "search"
+doc_type: "resourcePageType"
+---
+
+# aggregationOption resource type
+
+Namespace: microsoft.graph
+
+Specifies which aggregations should be returned alongside the search results.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/aggregationoption.md:14]: Lines 3 and 14:
1. The description in these 2 lines should match. Remember this description is used by search engines to display results. 
2. If the beta topic has the same issue, please leverage this PR to fix it as well.

File: api-reference/v1.0/resources/aggregationoption.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Properties
+
+| Property     | Type        | Description |
+|:-------------|:------------|:------------|
+|field|String|Specifies the field in the schema of the specified entity type that aggregation should be computed on. Required.|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/aggregationoption.md:20]: Line 20: This description is not easy to understand for a global audience. Please consider readers that only know basic English. For instance, please [avoid passive voice](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=25523) and use simple verbs.

File: api-reference/v1.0/resources/aggregationoption.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+|size|Int32|The number of [searchBucket](searchBucket.md) resources to be returned. This is not required when the range is provided manually in the search request. Optional.|
+|bucketDefinition|[bucketAggregationDefinition](bucketaggregationdefinition.md)|Specifies the criteria to compute an aggregation. Optional.|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/aggregationoption.md:22]: Please order the properties alphabetically.

File: api-reference/v1.0/api/search-query.md [hunk 5]
```diff
@@ -162,6 +162,8 @@ Content-type: application/json
 - Search [calendar events](/graph/search-concept-events)
 - Search content in SharePoint and OneDrive ([files, lists and sites](/graph/search-concept-files))
 - Search [custom types (Graph Connectors)](/graph/search-concept-custom-types) data
+- [Sort](/graph/search-concept-sort) search results
+- Use [aggregations](/graph/search-concept-aggregations) to refine search results
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/search-query.md:168]: Do we have a topic for the searchEntity resource type? If we don't have one, please create one.

File: api-reference/v1.0/resources/bucketaggregationdefinition.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+author: "nmoreau"
+ms.prod: "search"
+doc_type: "resourcePageType"
+---
+
+# bucketAggregationDefinition resource type
+
+Namespace: microsoft.graph
+
+Specifies details to aggregate search results.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/bucketaggregationdefinition.md:14]: Lines 3 and 14: 
1. Both descriptions should match. 
2. Please avoid using passive voice. 
3. Remember to make as simple as possible for search engines. 

Please check if the beta topics have the same issue.

File: api-reference/v1.0/resources/bucketaggregationdefinition.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+## Properties
+
+| Property     | Type        | Description |
+|:-------------|:------------|:------------|
+|sortBy|bucketAggregationSortProperty| The possible values are `count` to sort by the number of matches in the aggregation, `keyAsString`to sort alphabeticaly based on the key in the aggregation, `keyAsNumber` for numerical sorting based on the key in the aggregation. Required.
+|isDescending|Boolean|`True` to specify the sort order as descending. The default is `false`, with the sort order as ascending. Optional.|
+|prefixFilter|String|A filter to define a matching criteria. The key should start with the specified prefix to be returned in the response. Optional.|
+|minimumCount|Int32|The minimum number of items that should be present in the aggregation to be returned in a bucket. Optional.|
+|ranges|[bucketAggregationRange](bucketaggregationrange.md) collection|Specifies the manual ranges to compute the aggregations. This is only valid for non-string refiners of date or numeric type. Optional.|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/bucketaggregationdefinition.md:24]: Please arrange these properties alphabetically.

File: api-reference/v1.0/resources/searchbucket.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+author: "nmoreau"
+ms.prod: "search"
+doc_type: "resourcePageType"
+---
+
+# searchBucket resource type
+
+Namespace: microsoft.graph
+
+Represents a container for one or more search results that share the same value for the entity field that aggregates them.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/searchbucket.md:14]: Lines 3 and 14: Similar observation. I like the description in line 14 better.

File: api-reference/v1.0/resources/searchbucket.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+Represents a container for one or more search results that share the same value for the entity field that aggregates them. 
+
+## Properties
+
+| Property     | Type        | Description |
+|:-------------|:------------|:------------|
+|key|String| The discrete value of the field that an aggregation was computed on.|
+|count|Int32| The approximate number of search matches that share the same value specified in the **key** property. Note that this number is not the exact number of matches.|
+|aggregationFilterToken|String| A token containing the encoded filter to aggregate search matches by the specific **key** value. To use the filter, pass the token as part of the **aggregationFilter** property in a **searchRequest** object, in the format **"{field}:\\"{aggregationFilterToken}\\""**. See an [example](/graph/search-concept-aggregation#example-2-apply-an-aggregation-filter-based-on-a-previous-request).|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/searchbucket.md:22]: Please order these properties alphabetically.

File: api-reference/v1.0/resources/sortproperty.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+author: "nmoreau"
+ms.prod: "search"
+doc_type: "resourcePageType"
+---
+
+# sortProperty resource type
+
+Namespace: microsoft.graph
+
+Represents the sort options to order search results.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sortproperty.md:14]: Lines 3 and 14: Please rephrase to: Indicates the order to sort search results.

File: changelog/Microsoft.MicrosoftSearch.json [hunk 11]
```diff
@@ -205,6 +205,138 @@
...
+      "Version": "beta",
+      "CreatedDateTime": "2021-08-12T08:21:04.628237Z",
+      "WorkloadArea": "People and workplace intelligence",
+      "SubArea": "Analytics"
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "b09706d4-7465-4d34-bc1a-533e2be0a566",
+          "ApiChange": "EnumType",
```
Comment (isvargasmsft) [inline changelog/Microsoft.MicrosoftSearch.json:239]: Line 239: Please use `Enum type`, with a space. Guidance here: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=25523

File: changelog/Microsoft.MicrosoftSearch.json [hunk 12]
```diff
@@ -205,6 +205,138 @@
...
+          "Description": "Added the **bucketAggregationSortProperty** enumeration type.",
+          "Target": "bucketAggregationSortProperty"
+        },
+        {
+          "Id": "b09706d4-7465-4d34-bc1a-533e2be0a566",
+          "ApiChange": "Member",
+          "ChangedApiName": "unknownFutureValue",
+          "ChangeType": "Change",
+          "Description": "Removed the **unknownFutureValue** resource type",
+          "Target": "entityType"
```
Comment (isvargasmsft) [inline changelog/Microsoft.MicrosoftSearch.json:251]: Please clarify this entry. What are you deleting from where exactly? It is not clear.

File: changelog/Microsoft.MicrosoftSearch.json [hunk 13]
```diff
@@ -205,6 +205,138 @@
...
+        },
+        {
+          "Id": "b09706d4-7465-4d34-bc1a-533e2be0a566",
+          "ApiChange": "ComplexType",
```
Comment (isvargasmsft) [inline changelog/Microsoft.MicrosoftSearch.json:255]: ApiChange values for ComplexTypes should be Resource. Please make the change in the other entries as well.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Thank you very much for your work on this PR. 

Please take a look at my comments and let me know if you have any questions. 

Kind regards, 

Isaac Vargas

File: api-reference/v1.0/resources/aggregationoption.md (hunk 2 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/aggregationoption.md:14]: Hi @isvargasmsft, sorry, I lost the permission for editing the branches I created in this repo, I forked a new repo and created a new pull request for it. Please help have a look. This comment fixed in the new PR.
https://github.com/microsoftgraph/microsoft-graph-docs/pull/14159

File: api-reference/v1.0/resources/aggregationoption.md (hunk 3 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/aggregationoption.md:20]: This line changed to: "Computes aggregation on the field while the field exists in current entity type. Required."

File: api-reference/v1.0/resources/aggregationoption.md (hunk 4 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/aggregationoption.md:22]: Resolved.

File: api-reference/v1.0/resources/aggregationoption.md (hunk 2 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/aggregationoption.md:14]: Please help leave new comments in the #14159, I will abandon this PR in the future.

File: api-reference/v1.0/api/search-query.md (hunk 5 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/api/search-query.md:168]: Hi @isvargasmsft , little confusing about it. do you mean I need a new md file to introduce the searchEntity and link here?
Actually the detail description for these search entities were introduced in the search-api-overviw. Please refer below:
https://docs.microsoft.com/en-us/graph/api/resources/search-api-overview?view=graph-rest-1.0

File: api-reference/v1.0/resources/bucketaggregationdefinition.md (hunk 6 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/bucketaggregationdefinition.md:14]: Resolved, and also make same change in beta documents

File: api-reference/v1.0/resources/bucketaggregationrange.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
+---
+title: "bucketAggregationRange resource type"
+description: "Enables to specify some manual ranges in the aggregation request. This in only applicable to non string refiners : numeric and dates."
```
[the remark this answers is not in the document]
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/bucketaggregationrange.md:3]: Resolved, and also make same change in beta documents. keep the line 14.

File: api-reference/v1.0/resources/bucketaggregationrange.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+author: "nmoreau"
+ms.prod: "search"
+doc_type: "resourcePageType"
+---
+
+# bucketAggregationRange resource type
+
+Namespace: microsoft.graph
+
+Specifies the lower and upper limit to a range for aggregating search results. Applies to only refiners of the date or numeric type.
```
[the remark this answers is not in the document]
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/bucketaggregationrange.md:14]: Resolved, and also make same change in beta documents.

File: api-reference/v1.0/resources/searchbucket.md (hunk 8 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/searchbucket.md:14]: Resolved, and also make same change in beta documents. Keep the line 14.

File: api-reference/v1.0/resources/searchbucket.md (hunk 9 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/searchbucket.md:22]: Resolved

File: changelog/Microsoft.MicrosoftSearch.json (hunk 11 above)
Comment (yiwenwang-ms) [reply changelog/Microsoft.MicrosoftSearch.json:239]: Resolved

File: changelog/Microsoft.MicrosoftSearch.json (hunk 12 above)
Comment (yiwenwang-ms) [reply changelog/Microsoft.MicrosoftSearch.json:251]: Actually, the change log generated by the pipeline build tools, and the unknownFutureValue is a predefined value which mapped to unknownfuturevalue.  Correct the unknownfuturevalue to unknownFutureValue directly, so the change log appeared. I removed it from our change logs.

File: changelog/Microsoft.MicrosoftSearch.json (hunk 13 above)
Comment (yiwenwang-ms) [reply changelog/Microsoft.MicrosoftSearch.json:255]: Resolved.

File: api-reference/v1.0/resources/sortproperty.md (hunk 10 above)
Comment (yiwenwang-ms) [reply api-reference/v1.0/resources/sortproperty.md:14]: thank you, resolved.

**Comment by yiwenwang-ms** [comment]
Instead by https://github.com/microsoftgraph/microsoft-graph-docs/pull/14159

**Pull request closed without merging**